### PR TITLE
Tenative fix to let ruby 1.0.1 interop tests run

### PR DESCRIPTION
**Pull request by apolcyn**

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile [hunk 1]
```diff
@@ -70,12 +70,14 @@ RUN apt-get update && apt-get install -y time && apt-get clean
 RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
 RUN \curl -sSL https://get.rvm.io | bash -s stable
 
-# Install Ruby 2.1
-RUN /bin/bash -l -c "rvm install ruby-2.1"
-RUN /bin/bash -l -c "rvm use --default ruby-2.1"
+# Install Ruby 2.1.0
+# Workaround for https://github.com/grpc/grpc/issues/8905 to let the
+# interop tests run.
+RUN /bin/bash -l -c "rvm install ruby-2.1.0"
```
Comment (adelez) [inline tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile]: What's difference btw 2.1 and 2.1.0?

Comment (apolcyn) [reply tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile]: chatted offline, but FTR this should keep ruby-2.1.0 installed

**Comment by apolcyn** [comment]
there's an issue with running the interop tests for this PR because of what looks like an issue in oyulling down the docker image (ocker: Error response from daemon: pull access denied for grpctesting/grpc_interop_ruby_49cdb6452fe2dc7e3bea90a1caad29d2bdb52d11, repository does not exist or may require 'docker login'.)

**Comment by adelez** [comment]
@nicolasnoble : this is to build backwards comp image for v1.0.1 ruby. Do you think it should be checked in?

**Comment by apolcyn** [comment]
FTR this PR might be a bit strange to check in here. 

E.g., note that 
`git checkout v1.0.1 && tools/run_tests/run_interop_tests.py -l ruby -s ruby --use_docker` will still be broken...

My reason for checking this in here is only for history, so that we can build the image and then use it as something to successfully run the interop tests with the ruby-v1.0.1 release.

Since we can't move the `v1.0.1` tag, another alternative might be to just take the tweaked docker image here and keep it checked in somewhere else.

**Comment by adelez** [comment]
In this case, I vote for option2: building the tweeked image and not check in in 1.0.1. By the way, by checked in somewhere else you meant in master, right?

**Comment by apolcyn** [comment]
closing this in favor of either documenting the tweak and not checking it in, https://github.com/grpc/grpc/pull/13290, or checking in the tweaked image somewhere else in master

**Pull request closed without merging**